core/cli/src/params.rs

```diff
@@ -227,63 +225,81 @@ pub struct TransactionPoolParams {
 pub struct ExecutionStrategies {
 	/// The means of execution used when calling into the runtime while syncing blocks.
 	#[structopt(
-		long = "syncing-execution",
+		long = "execution-syncing",
 		value_name = "STRATEGY",
 		raw(
 			possible_values = "&ExecutionStrategy::variants()",
 			case_insensitive = "true",
 			default_value = r#""NativeElseWasm""#
 		)
 	)]
-	pub syncing_execution: ExecutionStrategy,
+	pub execution_syncing: ExecutionStrategy,
 
 	/// The means of execution used when calling into the runtime while importing blocks.
 	#[structopt(
-		long = "importing-execution",
+		long = "execution-import-block",
 		value_name = "STRATEGY",
 		raw(
 			possible_values = "&ExecutionStrategy::variants()",
 			case_insensitive = "true",
 			default_value = r#""NativeElseWasm""#
 		)
 	)]
-	pub importing_execution: ExecutionStrategy,
+	pub execution_import_block: ExecutionStrategy,
 
 	/// The means of execution used when calling into the runtime while constructing blocks.
 	#[structopt(
-		long = "block-construction-execution",
+		long = "execution-block-construction",
 		value_name = "STRATEGY",
 		raw(
 			possible_values = "&ExecutionStrategy::variants()",
 			case_insensitive = "true",
 			default_value = r#""Wasm""#
 		)
 	)]
-	pub block_construction_execution: ExecutionStrategy,
+	pub execution_block_construction: ExecutionStrategy,
 
 	/// The means of execution used when calling into the runtime while using an off-chain worker.
 	#[structopt(
-		long = "offchain-worker-execution",
+		long = "execution-offchain-worker",
 		value_name = "STRATEGY",
 		raw(
 			possible_values = "&ExecutionStrategy::variants()",
 			case_insensitive = "true",
-			default_value = r#""NativeWhenPossible""#
+			default_value = r#""Native""#
 		)
 	)]
-	pub offchain_worker_execution: ExecutionStrategy,
+	pub execution_offchain_worker: ExecutionStrategy,
 
 	/// The means of execution used when calling into the runtime while not syncing, importing or constructing blocks.
 	#[structopt(
-		long = "other-execution",
+		long = "execution-other",
 		value_name = "STRATEGY",
 		raw(
 			possible_values = "&ExecutionStrategy::variants()",
 			case_insensitive = "true",
-			default_value = r#""Wasm""#
+			default_value = r#""Native""#
+		)
+	)]
+	pub execution_other: ExecutionStrategy,
+
+	/// The execution strategy that should be used by all execution contexts.
+	#[structopt(
+		long = "execution-all",
```

Comment:
maybe just call this one `--execution` since it conflicts with all the other ones.